### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
## In this project milestone I accomplished the following:

[Live version](https://hamzaoutdoors.github.io/math-magicians/)

- [x] Deleted all the boilerplate from CRA (text, images, styles).
- [x] Create a directory called components.
- [x]  Created a new Calculator.js file.
 - [x] Created a React component that matches this design
 - [x] Develop just the UI of the calculator
- [x]  Used class-based components.
- [x]  The browser it render without problems.

_Thanks in advance for your feedback ❤️_

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-      </header>
+    <div className="app">
+      <img
+        src={logo}
+        alt="Logo"
+        className="logo"
+      />
+      <div className="app-container">
+        <h2 className="app-header">Let's do some math!</h2>
+        <Calculator />
```
Comment (oxenprogrammer) [inline src/App.js]: **Changes Required**
Good effort with the function component, however;
- [x] The `App Component` should be a `Class Component` as specified in the project requirement. Let us refactor it into a class component.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors , 

Good job so far!, I must applaud your efforts.
**To Highlight**
- [x] Elegant UI.
- [x] Excellent README file.
- [x] Live Demo Working.
- [x] Correct Gitflow.

   
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (Hamzaoutdoors) [reply src/App.js]: > Issue resolved .. thank you for your time and feedback

**Review by oxenprogrammer — APPROVED** [review]
Hi @Hamzaoutdoors , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**